### ISSUE TITLE
Template mangled smiley style

### ISSUE THREAD

**Issue by Thorarin** [axlevxa/typowiki#13]
It appears this template applies some styling using the CSS selector `.usr-content .page p img`, which unfortunately also matches some things that aren't regular images.

Things like the `FIXME ` macro or smileys/emoji use images, for example:
```html
<img src="https://mywiki/lib/images/smileys/fixme.svg" class="icon smiley" alt="FIXME">
<img src="https://mywiki/lib/images/smileys/cool.svg" class="icon smiley" alt="8-)">
```

Instead of appearing inline, these now appear huge, because they get stretched to full page width.

I locally added some custom CSS to override this behavior, which seems to do alright so far:
```css
.usr-content .page p img.icon {
  height: 1.2em;
  display: inline-block;
  margin: 0;
}
```

Probably some vertical alignment change is also warranted, but I haven't done that yet myself.

**Comment by axlevxa** [comment]
good catch - I'll try to fix that soon.

**Comment by axlevxa** [comment]
Fixed with release v1.3
